### PR TITLE
fix(styles): removes unwanted margin for heading in layouts/portal

### PR DESCRIPTION
**Pull request by b1aserlu**

### CODE REVIEW

**Review by imagoiq — CHANGES_REQUESTED** [review]
Thanks for the proposal, but it's only a workaround. The issue is that the styles are different between the index and intranet bundles (see https://next.design-system.post.ch/?path=/docs/getting-started-styles--docs). Can you please check with Philipp? Or assign the issue to him. Last time that I talked about this issue, he had some ideas.

**Comment by b1aserlu** [comment]
Asigned Issue to Phillip

**Comment by gfellerph** [comment]
See https://github.com/swisspost/design-system/issues/1931#issuecomment-1948478202. A temporary resolution to this problem would be my preferred way of dealing with it.

File: packages/styles/src/layouts/portal/_type.scss [hunk 1]
```diff
@@ -53,6 +53,9 @@
   h5,
   .h5 {
     margin-top: map.get(spacing.$post-sizes, 'bigger-big');
+    &:first-child {
+      margin-top: 0;
+    }
```
Comment (gfellerph) [inline packages/styles/src/layouts/portal/_type.scss:58]: This selector is too generic, scope it to the notification overlay title, no matter what heading level.

File: .changeset/spotty-countries-smoke.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': patch
+---
+
+Removes unwanted margin from heading when using `@use "@swisspost/design-system-styles/index.scss";`
```
Comment (gfellerph) [inline .changeset/spotty-countries-smoke.md:5]:
```suggestion
Removes unwanted margin from heading within the notification overlay component.
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

**Pull request merged**